vendor/github.com/getsentry/.gitignore

```diff
@@ -0,0 +1,3 @@
+# ... except for our self-symlink
```

Comment:
I'd add vendor/ to the .gitignore

Reply:
I have vendor/.gitignore specify `*`, equivalently.